files/cluster_provision.yml

```diff
@@ -66,34 +66,41 @@
                                 kernel_opts="ip=dhcp text inst.stage2=http://releases.linaro.org/reference-platform/enterprise/18.06/centos-installer/ inst.repo=http://mirror.centos.org/altarch/7/os/aarch64/ inst.ks=file:/ks.cfg"
                         fi
 
-                        # Chose known good kernel/initrd/cmdline
-
-                        # Build trigger machine_provision job
-                        cat << EOF > machine_provision
-                        machine_list=${machine_list}
-                        os_type=${os_type}
-                        machine_type=${machine_type}
+                        # Defaults
+                        machine_list="$( echo "$machine_list"| tr -d '[:space:]')"
+                        machine_name=${machine_list}
+                        machine_arch=AArch64
+                        machine_subarch=Grub
                         job_type=ohpc
+
+                        kernel_opts="${kernel_opts} earlycon console=ttyAMA0,115200"
```

Comment:
There are more common parts than just these two. You can create variables for stage2 and repo and set them in the if/else above, then set kernel opts once after.